indico/modules/users/util.py

```diff
@@ -422,7 +422,26 @@ def merge_users(source, target, force=False):
     # Restore the source user's primary email
     source.email = primary_source_email
     db.session.flush()
+    
+    log_dict = {
+        'module': 'User',
+        'message': 'User merged',
```

Comment:
Thinking about this again, instead of logging the first/last name in the data dict, you could also use log messages like this:

`f'User merged into {target.full_name}'`
`f'User merged from {source.full_name}'`